File: src/test/java/com/christophsens/cvenhancer/prompt/ImprovePromptTest.java
package com.christophsens.cvenhancer.prompt;

import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;
import org.springframework.ai.chat.prompt.Prompt;

public class ImprovePromptTest {

    @Test
    void testTranslationPrompt() {
        String expectedResult = """
            Improve this cv: "cv".
            Your response must be in the language of the CV.
            """;
        Prompt prompt = ImprovePrompt.getPrompt("cv");
        String result = prompt.getContents();
        Assertions.assertEquals(expectedResult, result);
    }
}

File: src/test/kotlin/com/christophsens/cvenhancer/prompt/ImprovePromptTest.kt
package com.christophsens.cvenhancer.prompt

import org.junit.jupiter.api.Assertions.assertEquals
import org.junit.jupiter.api.Test

class ImprovePromptTest {

    @Test
    fun testGetPrompt() {
        val cv = "Sample CV content"
        val contents = ImprovePrompt.getPrompt(cv).contents
        assertEquals("Improve this cv: \"${cv}\".\nYour response must be in the language of the CV.", contents)
    }
}